driver/sccb.c

```diff
@@ -40,16 +63,15 @@ static uint8_t ESP_SLAVE_ADDR   = 0x3c;
 
 int SCCB_Init(int pin_sda, int pin_scl)
 {
-    ESP_LOGI(TAG, "pin_sda %d pin_scl %d\n", pin_sda, pin_scl);
-    //log_i("SCCB_Init start");
-    i2c_config_t conf;
-    memset(&conf, 0, sizeof(i2c_config_t));
```

Comment:
above two lines ensure that the whole structure is created and that also all properties that will not be set are initialized as 0x00. How did you end up needing to change them?